### PR TITLE
First working base-structure

### PR DESCRIPTION
**Pull request by NilsNeeb**
Created first working base-structure

### CODE REVIEW

File: models/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class meeting__scheduler(models.Model):
```
Comment (szumbrunn) [inline models/models.py]: @NilsNeeb Don't use two successive underscores in object names. The odoo code convention is snake case

File: models/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    _name = 'meeting__scheduler.meeting__scheduler'
```
Comment (szumbrunn) [inline models/models.py]: @NilsNeeb You can call the object e.g. "meeting.scheduler" that's totally fine, then you don't have that repetition

File: models/models.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    _description = 'meeting__scheduler.meeting__scheduler'
+
+    name = fields.Char()
```
Comment (szumbrunn) [inline models/models.py]: @NilsNeeb Always set a label to fields: fields.Char(string="set label like this") then those can be translated later on. You can actually use german for the labels

File: demo/demo.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            <field name="name">Object 3</field>
+            <field name="value">30</field>
+          </record>
+
+          <record id="object4" model="meeting_scheduler">
+            <field name="name">Object 4</field>
+            <field name="value">40</field>
+          </record>
+    </data>
+</odoo>
```
Comment (SandroCalce) [inline demo/demo.xml]: No newline at end of file

File: controllers/__init__.py [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from . import controllers
```
Comment (SandroCalce) [inline controllers/__init__.py]: No newline at end of file

File: models/__init__.py
Comment (SandroCalce) [inline models/__init__.py:1]: No newline at end of File

File: __init__.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import controllers
+from . import models
```
Comment (SandroCalce) [inline __init__.py]: No newline at end of file

File: models/models.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    _name = 'meeting_scheduler'
+    _description = 'meeting_scheduler'
+
+    name = fields.Char(string="Your Name")
+    start_date = fields.Datetime(string="Start Date")
+    end_date = fields.Datetime(string="End Date")
+    location = fields.Char(string="Location")
+    subject = fields.Text(string="Subject")
+    duration = fields.Char(string="Duration", compute="_calc_duration", store=True)
+    frequency = fields.Selection([('0', 'Not repeating'), ('1', 'Weekly'), ('2', 'Biweekly')], store=True,
```
Comment (Lvk-v) [inline models/models.py]: describe variable names more expressive, for example meetingName, meetingStartDate, meetingLocation ...

File: security/ir.model.access.csv
Comment (SandroCalce) [inline security/ir.model.access.csv:1]: No newline at end of file

File: views/views.xml
Comment (SandroCalce) [inline views/views.xml:1]: No newline at end of file

**Review by SandroCalce — CHANGES_REQUESTED** [review]
Please add newlines to all the files mentioned

**Review by Lvk-v — CHANGES_REQUESTED** [review]
see models/models.py comments

**Review by Lvk-v — APPROVED** [review]

**Review by SandroCalce — APPROVED** [review]

**Pull request merged**